src/library/scala/collection/Seq.scala

```diff
@@ -225,6 +225,7 @@ trait SeqOps[+A, +CC[_], +C] extends Any
   /** Returns new $coll with elements in reversed order.
    *
    *  $willNotTerminateInf
+   *  $willForceEvaluation
```

Comment:
I could conceive of a lazy `IndexedSeq` with a `reversed` flag, but it's probably not worth specifying for that